packages/react/src/backlink/Backlink.tsx

```diff
@@ -0,0 +1,39 @@
+import { forwardRef, type Ref } from 'react';
+import { cx } from 'cva';
+import { Link as RACLink, type LinkProps } from 'react-aria-components';
+import { ArrowLeft } from '@obosbbl/grunnmuren-icons-react';
+
+type BacklinkProps = {
+  /** Additional CSS className for the element. */
+  className?: string;
+
+  /** Additional style properties for the element. */
+  style?: React.CSSProperties;
+
+  /** The URL to navigate to when clicking the breadcrumb. */
+  href?: string;
+
+  /** Icon styling */
+  iconStyle?: string;
+} & Omit<LinkProps, 'className' | 'style'>;
```

Comment:
Hvorfor blir `className` og `style` omitted her for å så legges inn på nytt inne i `BacklinkProps`?

Reply:
Egentligen osäker, har bara följt det mönstret som var i andra komponenter..